contrib/crane/template.py

```diff
@@ -0,0 +1,43 @@
+pkgname = "crane"
+pkgver = "0.20.1"
+pkgrel = 0
+build_style = "go"
+make_build_args = [
+    "./cmd/crane",
+]
+hostmakedepends = [
+    "go",
+]
+pkgdesc = "Container image and registry manipulation tool"
+maintainer = "Radosław Piliszek <radek@piliszek.it>"
+license = "Apache-2.0"
+url = "https://github.com/google/go-containerregistry"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "4bae53f34011e35ef874a60123b8ae70a5e992d804decb030479dbb888afe6d1"
+
+
+_supported_shells = ["bash", "zsh", "fish"]
+
+
+# NOTE(yoctozepto): docs are present but they do not render properly
```

Comment:
overly verbose comment, also we don't do the `NOTE(whatever)` stuff since it's practically useless

Reply:
ok, note it is useful after refactorings when `git blame` stops being that useful; I will also make it more succinct

Reply:
the chance of that happening and mattering is zero

Reply:
I'm not entirely convinced on the happening probability but mattering is probably low; I have just explained why I am trained to leave this kind of extras with NOTEs, agreed to remove